index.js

```diff
@@ -16,6 +16,77 @@ operations.deviation = deviation;
 operations.variance = variance;
 operations.count = count;
 
+/**
+* Takes a set of polygons, a set of points, and an array of aggregations, then performs them. Sum, average, count, min, max, and deviation are supported.
+*
+* @module turf/aggregate
+* @param {FeatureCollection} polygonFC - a GeoJSON FeatureCollection containing Polygons
+* @param {FeatureCollection} pointFC - a FeatureColelction containing Points
+* @param {Array} aggregations - an array of aggregation objects
+* @return {FeatureCollection} polys - a FeatureCollection containing polygons with specified aggregation statistics as attributes
+* @example
+* var aggregate = require('turf-aggregate')
+* var point = require('turf-point')
+* var polygon = require('turf-polygon')
+*var featurecollection =  require('turf-featurecollection')
+*
+* var poly1 = polygon([[[0,0],[10,0],[10,10],[0,10]]])
```

Comment:
rewrite `polygon` to `turf.polygon` and the same with `point` and `featurecollection` calls

Also this polygon is invalid: should be:

```js
var poly1 = turf.polygon([[[0,0],[10,0],[10,10],[0,10],[0,0]]])
```